solutions/CF Garland.mdx

```diff
@@ -22,33 +133,19 @@ $$\texttt{dp}[i+1][j+1][k] = \max(\texttt{dp}[i+1][j+1][k], \texttt{dp}[i][j][k]
 2. We append the character, but since it is the same as our previous character we get to "save" a paint.<center>$$\texttt{dp}[i+1][j][k] = \max(\texttt{dp}[i+1][j][k], \texttt{dp}[i][j][k] + 1)$$</center>
 
 
-Since this only gives us the best possible score that ends with length $i$, we can just take the current best and translate it using $\texttt{dp}[i+1][j][k] = \max(\texttt{dp}[i+1][j][k], \texttt{dp}[i][j][k])$
+Since this only gives us the best possible score that ends with length $i$, we can just take the current best and translate it using $\texttt{dp}[i+1][j][k] = \max(\texttt{dp}[i+1][j][k], \texttt{dp}[i][j][k])$.
 
-Now, we can query the max painting score for any substring containing the first character. However, this is not required; we only have to query the optimal paintings with size $n$ (the full string) with $\texttt{dp}[n][m_i][c_i]$
+Now, we can query the max painting score for any substring containing the first character. However, this is not required; we only have to query the optimal paintings with size $n$ (the full string) with $\texttt{dp}[n][m_i][c_i]$.
 
 <LanguageSection>
 
 <CPPSection>
 
+## C++ Implementation
```

Comment:
Should be `###`

(though why don't we just have one `### Implementation` above the languagesection instead of a separate `### [Language] Implementation` for each language?)

Reply:
sure but we usually to [language] impl right?

Reply:
Really? I see `### Implementation` used most of the time (though most of the time it's inside the LanguageSection, I think it would be better outside).

Reply:
ok